backend/src/api/acceleration/acceleration.routes.ts

```diff
@@ -3,7 +3,48 @@ import config from '../../config';
 import axios from 'axios';
 import logger from '../../logger';
 import mempool from '../mempool';
-import AccelerationRepository from '../../repositories/AccelerationRepository';
+import AccelerationRepository, { PublicAcceleration } from '../../repositories/AccelerationRepository';
+import { Acceleration } from '../services/acceleration';
+
+interface ApiAcceleration {
+  txid: string,
+  added: number,
+  status: 'completed' | 'requested' | 'accelerating' | 'failed',
+  effectiveFee: number,
+  effectiveVsize: number,
+  boostRate: number,
+  boostCost: number,
+  blockHeight: number | null,
+  pools: number[],
+}
+
+function mempoolAccelerationToApiAcceleration(mempoolAcceleration: Acceleration): ApiAcceleration {
```

Comment:
Would be good to clarify from `mempool.accelerations` to `services.accelerations` _(something like that)_ to avoid possible confusion in the future

Same bellow